Assets/Package/Runtime/Console/ConsoleGui.cs

```diff
@@ -144,14 +162,34 @@ private void EnsureInitialized()
             {
                 { inputSettings.ConfirmationKey, InvokeInputString },
                 { inputSettings.NextCommandKey, FetchNextCommand },
-                { inputSettings.PrevCommandKey, FetchPrevCommand }
+                { inputSettings.PrevCommandKey, FetchPrevCommand },
+                { inputSettings.AutocompleteKey, PerformAutocomplete }
             };
 
+            autocompleteHandler = new AutocompleteHandler();
             windowRect = styleSettings.InitialRect;
             scrollPosition = Vector2.zero;
             isInitialized = true;
         }
 
+        private void InputChanged()
+        {
+            UpdateAutocomplete();
+        }
+
+        private void UpdateAutocomplete()
+        {
+            if (string.IsNullOrEmpty(currentInput))
+            {
+                currentAutocompleteMatch = string.Empty;
+                return;
+            }
+
+            var splitInput = currentInput.Split(' ');
```

Comment:
Chciałbym zauważyć, że to chyba nie będzie działać dobrze, że apostrofami, masz już gotową logikę do splitowania inputu

Reply:
Good catch, trochę wstyd coś takiego przegapić. Poprawiłem.